src/spyglass/lfp_band/v1/lfp_band.py

```diff
@@ -195,7 +193,9 @@ def make(self, key):
         lfp_band_elect_id = lfp_band_elect_id[lfp_sort_order]
         lfp_band_ref_id = lfp_band_ref_id[lfp_sort_order]
 
-        lfp_sampling_rate = lfp_part_table.fetch1("filter_sampling_rate")
+        lfp_sampling_rate = LFPOutput.merge_get_parent(lfp_key).fetch1(
+            "lfp_sampling_rate"
+        )
```

Comment:
If you fetch both here, you get a minor time save of the extra of the extra fetch call
```suggestion
        lfp_sampling_rate, lfp_interval_list = LFPOutput.merge_get_parent(lfp_key).fetch1(
            "lfp_sampling_rate",  "interval_list_name"
        )
```